src/main/java/Group15/View/MuscleSelectionView.java

```diff
@@ -182,7 +182,7 @@ private static void createEquipmentCheckboxFunctionality(Button allButton, Butto
         noneButton.setOnAction(_ -> {
             for (Equipment equipment : Equipment.values()) {
                 if(selectedEquipment.contains(equipment)) {
-                    MuscleSelectionView.selectedEquipment.remove(equipment);
+                    MuscleSelectionView.selectedEquipment.clear();
```

Comment:
Why are you looping through all equipment values and check if they are in equipment, only to clear everything?

Reply:
.clear() removes everything from the list, and that is what you want the none button to do

Reply:
I went too fast hold on